src/printer.js

```diff
@@ -1996,6 +2000,8 @@ function printFunctionParams(path, print, options) {
 
   return concat([
     "(",
+    comments.printDanglingComments(path, options, /* sameIndent */ true),
```

Comment:
This is clearly an opinionated choice here. I try to stay consistent with the guidelines provided by @jlongster for regarding comments positioning in template literals.